_commands/tools/time.md

```diff
@@ -0,0 +1,119 @@
+---
+---
+
+time
+---
+`time` prints out the total time elapsed, the time used by system overhead and the time used to execute the utility.
+	
+<!--more-->
+
+### Situation
+
+If you simply want to output the time it takes to run a command
+	
+~~~ bash
+$ time ./project -m < input.txt
+~~~
+
+### Meaning:
+`time` calls the bash time command, `./project`is the program you are trying to run,
+`-m` is a flag for that program and `< input.txt `is the input file for the program.
+Anything that appears after `./project` is taken like it would be without using the time command.
+This will output the following:
+
+~~~ bash
+0.00user 0.00system 0:00.00elapsed 
```

Comment:
Gotcha. I ran it on my Mac and an Ubuntu 14.04 server and got the real/user/sys times, but if the class VM produces the output you pasted it's worth keeping. Will merge now, thanks for following up!